### PR TITLE
dm: support empty check for physical mode

### PR DESCRIPTION
**Pull request by okJiang**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tiflow/issues/11115

### What is changed and how it works?

- as title

#### Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

### Release note 

```release-note
Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

If you don't think this PR needs a release note then fill it with `None`.
```

### CODE REVIEW

File: dm/checker/checker.go [hunk 1]
```diff
@@ -533,12 +536,29 @@ func (c *Checker) Init(ctx context.Context) (err error) {
...
+			c.checkList = append(c.checkList, checker.NewLightningEmptyTableChecker(lChecker))
+		}
 	}
 
 	c.tctx.Logger.Info(c.displayCheckingItems())
 	return nil
 }
 
+func onlyTableEmptyCheck(checkingItems map[string]string) bool {
+	for _, item := range config.LightningPrechecks {
```
Comment (lance6716) [inline dm/checker/checker.go:553]: maybe rename the function to `lightningCheckGroupOnlyTableEmpty`

**Review by lance6716 — COMMENTED** [review]
rest lgtm

**Review by lance6716 — APPROVED** [review]

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**